src/components/custom/platforms-dropdown.js

```diff
@@ -20,10 +20,15 @@ const FadeInDown = keyframes`
     }
 `
 
+const NavDropdownWrapper = styled.div`
```

Comment:
```suggestion
```
This can be removed and be replaced with the `Flex` component. The `Flex` component have `display: flex;`, `justify-content: center;` and `align-items: center;` by default

Reply:
Good call, replaced.